src/VisualStudio/Core/Def/DocumentOutline/DocumentOutlineViewModel_Utilities.cs

```diff
@@ -156,72 +161,39 @@ static bool Contains(LspDocumentSymbol parent, LspDocumentSymbol child)
             // Converts a Document Symbol Range to a SnapshotSpan within the text snapshot used for the LSP request.
             SnapshotSpan GetSymbolRangeSpan(Range symbolRange)
```

Comment:
It seems surprising to me if there isn't a helper for this somewhere. @dibarbet?